indico/htdocs/js/indico/modules/eventcreation.js

```diff
@@ -0,0 +1,62 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function(global) {
+    'use strict';
+
+    global.setupEventCreationDialog = function setupEventCreationDialog(options) {
+        options = $.extend({
+            categoryField: null,
+            protectionModeFields: null,
+            initialCategory: null
+        }, options);
+        var messages = $($.parseHTML($('#event-creation-protection-messages').html()));
```

Comment:
The messages are in a `script[type=text/html]` tag to keep them out of the normal DOM (since we don't want them to all show up at once). Poor man's templating basically